icons/citrus.svg

```diff
@@ -9,9 +9,8 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <path d="M5.51 18.49a12 12 0 0 0 16.12.78c.49-.41.49-1.15.03-1.6L6.34 2.33a1.08 1.08 0 0 0-1.6.03A12 12 0 0 0 5.5 18.5Z" />
-  <path d="M8.34 15.66a8 8 0 0 0 10.4.78c.54-.4.54-1.16.06-1.64L9.2 5.2c-.48-.48-1.25-.48-1.64.06a8 8 0 0 0 .78 10.4Z" />
-  <path d="m14 10-5.5 5.5" />
-  <path d="M14 10v8" />
-  <path d="M14 10H6" />
+  <path d="M5.5 1.5a12.02 12.02 0 0 0 17 17z" />
+  <path d="M8.5 4.5a7.78 7.78 0 1 0 11 11" />
+  <path d="M14 17.9V10l-5.6 5.63" />
+  <path d="M14 10H6.1" />
```

Comment:
The round corners are essentially a feature, not a bug, they make sure that the icon fits within the 1px padding box:
```suggestion
  <path d="M2 10a12 12 0 0 0 19.62 9.27c.5-.4.5-1.16.04-1.61L6.34 2.34c-.45-.45-1.2-.46-1.61.04A11.95 11.95 0 0 0 2 10Z" />
  <path d="M19.65 15.66A8 8 0 0 1 8.35 4.34" />
  <path d="m14 10-5.5 5.5" />
  <path d="M14 18v-8H6" />
```